configs/15.0/packages/glibc/stage_2

```diff
@@ -118,7 +118,19 @@ atcfg_configure() {
 
 # Make build command
 atcfg_make() {
-	PATH=${at_dest}/bin:${PATH} ${SUB_MAKE}
+	if [[ ("${distro_fm}" == "ubuntu") || ("${distro_fm}" == "debian") ]]; then
+			if [[ "${build_arch}" == "ppc64le" ]]; then
+					user_defined_dir='user-defined-trusted-dirs=/lib/powerpc64le-linux-gnu:/usr/lib/powerpc64le-linux-gnu'
+			elif [[ "${build_arch}" == "ppc64" ]]; then
+					user_defined_dir='user-defined-trusted-dirs=/lib/powerpc64-linux-gnu:/usr/lib/powerpc64le-linux-gnu'
```

Comment:
There is a typo in this line: should be `/usr/lib/powerpc64-linux-gnu` instead of `/usr/lib/powerpc64le-linux-gnu`.

But AT does not support ppc64 (BE) anymore, so I don't think there's a need to add it here. It would be even hard to test it properly.